providers/ms365/resources/serviceprincipals.go

```diff
@@ -16,6 +16,11 @@ import (
 	"go.mondoo.com/cnquery/v11/utils/stringx"
 )
 
+const (
+	MicrosoftEntraTenantID = "f8cdef31-a31e-4b4a-93e4-5f571e91255a"
+	MicrosoftTenantID      = "72f988bf-86f1-41af-91ab-2d7cd011db47"
```

Comment:
can we add some doc here? unclear what those ids are

Reply:
Comment added